.travis.yml

```diff
@@ -7,6 +7,7 @@ sudo: false
 
 install:
   - go get github.com/russross/blackfriday
+  - go get github.com/shurcooL/github_flavored_markdown
   - go get github.com/PuerkitoBio/goquery
   - go get github.com/gorilla/mux
   - go get github.com/microcosm-cc/bluemonday
```

Comment:
@shurcooL I personally do that too, but it may be used to see project's dependencies list, so I just leave it as is.